float32 yaw
float32 pitch
float32 roll
float32 yaw_velocity
float32 pitch_velocity
float32 roll_velocity
